### DISCUSSION TITLE
case.wt causes error

### DISCUSSION THREAD

**Discussion by li-xuyang28** [kogalur/randomForestSRC#302] [General]
When trying to apply case weight (which is a column of the input data.frame, that is of type numeric), the following error occur:

Error in get.weight(case.wt, subj.unique.count) : 
  'list' object cannot be coerced to type 'double'

If I check the input column, it is indeed numeric

> class(data.rfsrc$weights)
[1] "numeric"

There is no missing value and it runs normally without class.wt.

**Comment by ishwaran** [comment]
I do not understand why you are using the `get.weight` function which is an internal function and not typically used.

What exactly are you trying to accomplish?  Why is case weight a column of the input data frame?  Please provide a worked out example in order for us to help you.
